vendor/foxy/CMakeLists.txt

```diff
@@ -153,6 +158,13 @@ target_link_libraries(
         certify::core
 )
 
```

Comment:
The original `djarek/certify` adds `CoreFoundation` and `Security` on to the linker explicitly. This fork does not.